backend/internal/models/event.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// Represents Venue Event
+type Event struct {
+	EventID   int       `json:"event_id"`
+	Name      string    `json:"name"`
+	Date      time.Time `json:"event_date"`
+	Time      time.Time `json:"event_time"`
+	ImagePath string    `json:"image_path"`
+	VenueID   uuid.UUID `json:"venue_id"`
+	CreatedAt time.Time `json:"created_at"`
+}
+
+func (e *Event) Validate() map[string]string {
+	return nil

```

Comment:
- [ ] fill out this stub. probably want simple checks on string len of [name, image path], etc etc based on requirements